payments-core/src/test/java/com/stripe/android/payments/PaymentFlowFailureMessageFactoryTest.kt

```diff
@@ -115,6 +118,13 @@ class PaymentFlowFailureMessageFactoryTest {
         )
 
         val result = factory.create(intent, StripeIntentResult.Outcome.FAILED)
-        assertThat(result).isEqualTo("this is a message that should be shown")
+        assertThat(result).isEqualTo("Your card was declined")
     }
 }
+
+private fun withLocale(locale: Locale, block: () -> Unit) {
+    val original = Locale.getDefault()
+    Locale.setDefault(locale)
+    block()
+    Locale.setDefault(original)
```

Comment:
You probably want this in a `finally` block. Otherwise one test could cause later tests to fail.